WordPressKitTests/TransactionsServiceTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+import WordPressShared
+import CocoaLumberjack
+@testable import WordPressKit
+
+class TransactionsServiceTests: RemoteTestCase, RESTTestable {
+    
+    let supportedCountriesSuccessFileName = "supported-countries-success.json"
+    var remote: TransactionsServiceRemote!
+
+    override func setUp() {
+        super.setUp()
+        remote = TransactionsServiceRemote(wordPressComRestApi: getRestApi())
+    }
+    
+    func testGetSupportedCountries() {
+        let expect = expectation(description: "Get supported countries success")
+        
+        stubRemoteResponse("me/transactions/supported-countries/",
+                           filename: supportedCountriesSuccessFileName,
+                           contentType: .ApplicationJSON,
+                           status: 200)
+        
+        remote.getSupportedCountries(success: { (countryList) in
+            expect.fulfill()
+            XCTAssert(countryList.count == 239)
```

Comment:
This seems awfully fragile. If the API adds or removes a supported country, the test will fail. I'm not sure that's a good reason for it to. Is that what we really want to happen?

Reply:
service response is mocked on the client side so as soon as nobody changes the mock data it won't fail. Actually I don't see any other way. I need to check if the response parsing is successful, do you have any other suggestions?